src/sentry/utils/cache.py

```diff
@@ -1,44 +1,14 @@
-from collections.abc import Callable, Mapping
-from typing import Any, Generic, TypeVar
+from collections.abc import Mapping
+from typing import Any, TypeVar
 
 from django.core.cache import cache
 
-__all__ = ["cache", "memoize", "default_cache", "cache_key_for_event"]
+__all__ = ["cache", "default_cache", "cache_key_for_event"]
 
 default_cache = cache
 
 T = TypeVar("T")
 
 
-class memoize(Generic[T]):
```

Comment:
I think you'll have to do `getsentry` first -- it looks like it uses this as well

Reply:
True - https://github.com/getsentry/getsentry/pull/13419